### ISSUE TITLE
Gray Screen when logging in; likely a cookie issue

### ISSUE THREAD

**Issue by PatrickLiu2000** [gt-scheduler/website#35]
There is a gray Screen when opening up the app; it is likely a cookie issue.

<details>
<summary>Cookie Dump</summary>

```json
[
{
    "domain": ".gt-scheduler.org",
    "expirationDate": 1680996433,
    "hostOnly": false,
    "httpOnly": false,
    "name": "_ga",
    "path": "/",
    "sameSite": "unspecified",
    "secure": false,
    "session": false,
    "storeId": "0",
    "value": "GA1.1.394932681.1617642774",
    "id": 1
},
{
    "domain": ".gt-scheduler.org",
    "expirationDate": 1680996433,
    "hostOnly": false,
    "httpOnly": false,
    "name": "_ga_TPJQQ46MS9",
    "path": "/",
    "sameSite": "unspecified",
    "secure": false,
    "session": false,
    "storeId": "0",
    "value": "GS1.1.1617924433.3.0.1617924433.0",
    "id": 2
},
{
    "domain": "www.gt-scheduler.org",
    "expirationDate": 1743183297,
    "hostOnly": true,
    "httpOnly": false,
    "name": "202108",
    "path": "/",
    "sameSite": "unspecified",
    "secure": false,
    "session": false,
    "storeId": "0",
    "value": "{%22desiredCourses%22:[%22CS%204641%22%2C%22CS%203630%22%2C%22CS%202110%22%2C%22CS%204510%22%2C%22SOC%201101%22]%2C%22pinnedCrns%22:[%2287568%22%2C%2290781%22%2C%2284968%22%2C%2280311%22%2C%2289264%22]%2C%22excludedCrns%22:[]%2C%22colorMap%22:{%22CS%204641%22:%22#333333%22%2C%22CS%203630%22:%22#B0BC00%22%2C%22CS%202110%22:%22#808080%22%2C%22CS%204510%22:%22#AB149E%22%2C%22SOC%201101%22:%22#9F0500%22}%2C%22sortingOptionIndex%22:0}",
    "id": 3
},
{
    "domain": "www.gt-scheduler.org",
    "expirationDate": 1744068433,
    "hostOnly": true,
    "httpOnly": false,
    "name": "term",
    "path": "/",
    "sameSite": "unspecified",
    "secure": false,
    "session": false,
    "storeId": "0",
    "value": "undefined",
    "id": 4
},
{
    "domain": "www.gt-scheduler.org",
    "expirationDate": 1647986106,
    "hostOnly": true,
    "httpOnly": false,
    "name": "visited-merge-notice",
    "path": "/",
    "sameSite": "unspecified",
    "secure": false,
    "session": false,
    "storeId": "0",
    "value": "true",
    "id": 5
}
]
```

</details>

**Comment by jazevedo620** [comment]
The issue is likely due to the fact that the `term` cookie has a value of `"undefined"` and the app does not handle such a scenario.

**Comment by jazevedo620** [comment]
I believe that this may be fixed by #23 [(relevant code)](https://github.com/gt-scheduler/website/pull/23/files#diff-df083a776c8512b421952bdfefa3f49d267924da306e81c8ca8659597330e6c7R184) once it is merged.

**Comment by jazevedo620** [comment]
This is fixed by #69
